scaleway/resource_k8s_cluster_beta.go

```diff
@@ -281,6 +339,28 @@ func resourceScalewayK8SClusterBetaCreate(d *schema.ResourceData, m interface{})
 
 	req.AutoscalerConfig = autoscalerReq
 
+	autoUpgradeEnable, okAutoUpgradeEnable := d.GetOk("auto_upgrade.0.enable")
+	autoUpgradeStartHour, okAutoUpgradeStartHour := d.GetOk("auto_upgrade.0.maintenance_window_start_hour")
+	autoUpgradeDay, okAutoUpgradeDay := d.GetOk("auto_upgrade.0.maintenance_window_day")
+
+	// check if either all or none of the auto upgrade attribute are set.
```

Comment:
Can you rework this comment, please? Lines 1, 2 and 4 of your comment say more or less the same things.

Reply:
@kindermoumoute https://github.com/terraform-providers/terraform-provider-scaleway/pull/376#discussion_r360577837 :sweat_smile:

Reply:
removed the 4th line, should be better

Reply:
removed the 4th line, should be better